App/src/Entity/User.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\OneToMany;
+
+#[ORM\Entity]
+#[ORM\Table(name: "`user`")]
+class User
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private int $id;
+
+    #[OneToMany(targetEntity: Expense::class, mappedBy: 'payer')]
+    private Collection|null $expenses = null;
```

Comment:
Pas besoin de rendre ta collection de dépenses optionnelles : dans le cas minimal tu auras une collection vide dans tous les cas.

```suggestion
    #[OneToMany(targetEntity: Expense::class, mappedBy: 'payer')]
    private Collection $expenses;
```